chapters/delaVarga.md

```diff
@@ -0,0 +1,163 @@
+```python
+gp.get_raw_data(geo_data, 'foliations').head()
+```
+
+<div>
+
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>X</th>
+      <th>Y</th>
+      <th>Z</th>
+      <th>azimuth</th>
+      <th>dip</th>
+      <th>polarity</th>
+      <th>formation</th>
+      <th>series</th>
+      <th>order_series</th>
+      <th>G_x</th>
+      <th>G_y</th>
+      <th>G_z</th>
+      <th>isFault</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>917.45</td>
+      <td>1000.0</td>
+      <td>-1135.398</td>
+      <td>270.0</td>
+      <td>71.565</td>
+      <td>1</td>
+      <td>MainFault</td>
+      <td>fault</td>
+      <td>1</td>
+      <td>-0.948683</td>
+      <td>-1.742702e-16</td>
+      <td>0.316229</td>
+      <td>True</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>1450.00</td>
+      <td>1000.0</td>
+      <td>-1150.000</td>
+      <td>90.0</td>
+      <td>18.435</td>
+      <td>1</td>
+      <td>Reservoir</td>
+      <td>Rest</td>
+      <td>2</td>
+      <td>0.316229</td>
+      <td>1.936342e-17</td>
+      <td>0.948683</td>
+      <td>False</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+We can visualize the 3D data using vtk with the following function.
+```python
+gp.visualize(geo_data)
+```
+![png](../figures/GemPyInputData.png)
+
+Now we can categorize the data into different depositional series and faults.
+```python
+gp.set_data_series(geo_data, {"fault":geo_data.formations[4], 
+                              "Rest":np.delete(geo_data.formations, 4)},
+                    order_series = ["fault", "Rest"])
+```
+And we are ready to interpolate:
+```python
+interp_data = gp.InterpolatorInput(geo_data, u_grade=[3,3])
+sol = gp.compute_model(interp_data)
+gp.visualize(geo_data, pot_field=sol[1,:])
+```
+
+![png](../figures/GemPyOutput.png)
+
+
+# 
+As any other open-source project, GemPy
+ did not start from scratch. Pandas for data management [2], the
+ Visualization Toolkit (vtk) for 3D visualization and manipulation [3],
+ skimage for computing topology and extract 3D surfaces [4] or the use
+  of  pynoddy for simulating kinematics [5]; are only an
+  example of how much can be accomplish integrating what the latest developments
+   of the scientific community in the open-source scene.
+
+ GemPy was born with 3D geological modelling as an inference problem
+  in mind. This means that automatic differentiation to compute gradients
+   and computational performance were key. To achieve this, GemPy is built
+   on top of Theano [6]. Theano is a Python library that allows to define
+   symbolically mathematical expressions to be optimized and differentiated.
+   If this were not enough the code is compile  either in C or CUDA for the use of
+   GPUs. The use of Theano opens up the integration with pymc3 [7] for a
+   fully Bayesian framework in geological modelling.
+
+ But this is only the beginning. Making geological modelling open-source
```

Comment:
Thank you @rowanc1. I have been a bit cautious  advertising GemPy too much yet because it is in active development. I hope to have an stable version and the paper out in a couple of months. And regarding to simpeg, yes I have been looking at your library for the geophysical inversions but the problem I have is that I need to have all the math written in Theano to get the automatic differentiation.

You are coming to Germany next month arent you?